### ISSUE TITLE
"illegal access"

### ISSUE THREAD

**Issue by BowuDev** [leops/fgdparser#1]
From my understanding the ever `illegal access` occurs when a jsonArray has a mishab, or json.stringify can not be completed...
I took into the account this error could happen for other reasons in `node.js`...
But it got patched back in ~v4!
I'm currently using `v6.10.0`!

This appeared while in node!
```javascript
> try{ F( "halflife2.fgd" ) } catch(E) { console.log(E) }
illegal access

>
```

I would just like a simple explanation if this is my side, or the actual code.

**Comment by leops** [comment]
The original 0.1.0 version was hastily written and only tested with Portal 2 FGDs.
I just published version 0.2.0, a version that is still hastily written but should now be compatible with Alien Swarm, CS:GO, GMOD, HL2, L4D2 and TF2.
Concerning the platform versions, the 0.1.0 release was written for and tested on Electron 1.6.4 (node 7.4.0), and version 0.2.0 has been additionally tested with node 7.7.1

**Comment by BowuDev** [comment]
Thank you very much!
